eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -101,13 +101,17 @@ jobs:
       - script: |
           node common/scripts/install-run-rush.js build -t "${{parameters.PackageName}}" --verbose -p max
         displayName: "Build libraries"
+        env:
+          RUSH_BUILD_CACHE_CREDENTIAL: $(js-rush-blob-build-cache-credential-readonly)
```

Comment:
Where is this variable set? Also why is this only for tests? Is it more intersting for testing than builds?

Reply:
The secrets come from a KV variable group. I want to enable cache writing in private pipelines for now.  It may be fine to have read access from public pipelines too. I want to be cautious so only doing this for live test pipelines which are private.

Reply:
Can you make the blob public so we don't need any tokens to read it? Then we only need to setup a secret token when we are writing it.

Reply:
Yes technically a blob with public anonymous read access works.  If it is not a violation of any policy on our Azure subscriptions we could do that.

Reply:
Nope we have several places where we host public content! @weshaggard do you have thoughts on a resource group location here?

Reply:
azuresdkartifacts is the container we use for other places. I think that would work for here as well. @jeremymeng I'll create a container and grant you permissions.